app/controllers/proposal_controller.rb

```diff
@@ -16,11 +16,13 @@ def show
   def new
     @user = User.new
     @url = conference_program_proposal_index_path(@conference.short_title)
+    @languages = @program.languages.split(',').map {|l| ISO_639.find(l).english_name} if @program.languages.present?
   end
 
   def edit
     authorize! :edit, @event
     @url = conference_program_proposal_path(@conference.short_title, params[:id])
+    @languages = @program.languages.split(',').map {|l| ISO_639.find(l).english_name} if @program.languages.present?
```

Comment:
That's exactly the same line code as in the new action ... so please DRY it in a function!